### PR TITLE
ccd_contam_file_error_message

### PR DESCRIPTION
**Pull request by joyvelasquez**
Creating an error message for CCD file without data for requested dates.

### CODE REVIEW

File: xrtpy/response/effective_area.py [hunk 1]
```diff
@@ -98,33 +100,47 @@ def observation_date(self, date):
...
             t0 = _ccd_contamination_file_time[0]
             dt = time - t0
+            ccd_data_dates_dt.append((epoch + timedelta(0, dt)))
             ccd_data_dates_to_seconds.append(
                 float((epoch + timedelta(0, dt)).strftime("%s"))
             )
 
+        if self.observation_date > ccd_data_dates_dt[-1]:
+            raise ValueError(
+                f"No contamination data is presently available for {self.observation_date}.\n The latest available data {ccd_data_dates_dt[-1]}.\n Contamination data is update monthly. Last update was on {self.xrt_contam_on_ccd_geny_update}"
```
Comment (namurphy) [inline xrtpy/response/effective_area.py:126]:
```suggestion
                "No contamination data is presently available for "
                f"{self.observation_date}.\n The latest available data "
                f"{ccd_data_dates_dt[-1]}.\n Contamination data is "
                "updated periodically. Last update was on "
                f"{self.xrt_contam_on_ccd_geny_update}. If this is more "
                "than one month ago, please raise an issue at: "
                "https://github.com/HinodeXRT/xrtpy/issues/new"
```

**Comment by joyvelasquez** [comment]
pre-commit.ci autofix

File: xrtpy/response/effective_area.py [hunk 2]
```diff
@@ -97,33 +100,53 @@ def observation_date(self, date):
...
 
+        if self.observation_date > ccd_data_dates_dt[-1]:
+            raise ValueError(
+                "No contamination data is presently available for "
+                f"{self.observation_date}.\n The latest available data "
+                f"{ccd_data_dates_dt[-1]}.\n Contamination data is "
+                "updated periodically. Last update was on "
+                f"{self.xrt_contam_on_ccd_geny_update}. If this is more "
+                "than one month ago, please raise an issue at: "
+                "https://github.com/HinodeXRT/xrtpy/issues/new"
```
Comment (namurphy) [inline xrtpy/response/effective_area.py:132]:
```suggestion
                "No contamination data is presently available for "
                f"{self.observation_date}.\n The latest available data is on "
                f"{ccd_data_dates_dt[-1]}.\n Contamination data is "
                "updated periodically. The last update was on "
                f"{self.xrt_contam_on_ccd_geny_update}. If this is more "
                "than one month ago, please raise an issue at: "
                "https://github.com/HinodeXRT/xrtpy/issues/new"
```

File: xrtpy/response/effective_area.py [hunk 3]
```diff
@@ -97,33 +100,53 @@ def observation_date(self, date):
...
+            )
         return ccd_data_dates_to_seconds
 
+    @property
+    def ccd_observation_date_to_seconds(self):
+        """Converting users observation date into seconds with respect to CCD contamination data. Used for interpolation."""
```
Comment (namurphy) [inline xrtpy/response/effective_area.py:138]:
```suggestion
        """Converting users observation date into seconds with
        respect to CCD contamination data. Used for interpolation."""
```

**Review by namurphy — APPROVED** [review]
LGTM!

**Pull request merged**